ui/app/templates/vault/cluster/secrets/backend/metadata.hbs

```diff
@@ -1,7 +1,7 @@
 <PageHeader as |p|>
   <p.top>
     <KeyValueHeader
-      @baseKey={{hash id=model.id}}
+      @baseKey={{hash id=this.id}}
```

Comment:
id is defined on the route, which is pulled from the params
`this.id = secret;`